src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -264,24 +267,24 @@ class OrganizationMetadataEntitySetsService(private val edmService: EdmManager)
                 propertyTypes.getValue(COL_INFO).id to setOf(mapper.writeValueAsString(column))
         )
 
-        val columnId = dataGraphManager.getEntityKeyIds(
+        val columnEntityKeyId = dataGraphManager.getEntityKeyIds(
                 setOf(EntityKey(organizationMetadataEntitySetIds.columns, "${column.tableId}.${column.id}"))
         ).first()
 
 
-        val id = dataGraphManager.getEntityKeyIds(
+        val datasetEntityKeyId = dataGraphManager.getEntityKeyIds(
                 setOf(EntityKey(organizationMetadataEntitySetIds.datasets, table.id.toString())) //TODO: Consider using OID
         ).first()
 
-        dataGraphManager.mergeEntities(
+        dataGraphManager.partialReplaceEntities(
                 organizationMetadataEntitySetIds.columns,
-                mapOf(id to columnEntity),
+                mapOf(columnEntityKeyId to columnEntity),
                 columnAuthorizedPropertTypes
         )
 
-        dataGraphManager.mergeEntities(
+        dataGraphManager.partialReplaceEntities(
```

Comment:
Needs to be merge otherwise there will only be one column in the array of properties.